oscap/rhel8/xccdf_org.ssgproject.content_profile_anssi_bp28_enhanced/customizations.json

```diff
@@ -0,0 +1 @@
+{"filesystem":[{"min_size":1073741824,"mountpoint":"/boot"},{"min_size":1073741824,"mountpoint":"/home"},{"min_size":1073741824,"mountpoint":"/opt"},{"min_size":1073741824,"mountpoint":"/srv"},{"min_size":1073741824,"mountpoint":"/tmp"},{"min_size":5368709120,"mountpoint":"/usr"},{"min_size":3221225472,"mountpoint":"/var"},{"min_size":5368709120,"mountpoint":"/var/log"},{"min_size":10737418240,"mountpoint":"/var/log/audit"},{"min_size":1073741824,"mountpoint":"/var/tmp"}],"packages":["aide","sudo","dnf-automatic","rsyslog-gnutls","chrony"]}
```

Comment:
Nitpick: it can we try add a new line at the end of the file?

Reply:
I'll have to check, because it's `json.Marshall` that is doing the writing. Is that possible with the tool ?

Reply:
Good question, you might be able to append `\n` to the buffer before writing to the file

Reply:
I've done that. Good idea thanks.